src/runners/junitRunner/JUnitRunnerResultAnalyzer.ts

```diff
@@ -109,9 +109,9 @@ export class JUnitRunnerResultAnalyzer extends BaseRunnerResultAnalyzer {
          * '(?:@AssumptionFailure: |@Ignore: )?' - indicate if the case is ignored due to assumption failure or disabled
          * '(.*?)'                               - test method name
          * '(?:\[\d+\])?'                        - execution index, it will appear for the JUnit4's parameterized test
-         * '\((.*?)\)'                           - class fully qualified name
+         * '\(([^)]*)\)[^(]*$'                   - class fully qualified name
```

Comment:
Add more comments here?

`class fully qualified name which wrapped by the last paired brackets, see: https://github.com/microsoft/vscode-java-test/issues/1075`